src/NewEmail/Form/index.jsx

```diff
@@ -0,0 +1,103 @@
+// Note for the next TEAM:1. For function's you have console.log() to see in console what is happening
+// Note for the next TEAM:2. In this Form component you will find The Form and the Buttons
+// Note for the next TEAM:3. You will find comments before Function for what are they and so on
+import {useState} from "react"
+function Form() {
+    const [email, setEmail] = useState("")
+    const [subject, setSubject] = useState("")
+    const [textArea, setTextArea] = useState("")
+    const [send, setSend] = useState(false)
+    const [cancel, setCancel] = useState(false)
+
+    // Function for cancel button ->
+    const handleCancel = () => {
+        setEmail("")
+        setSubject("")
+        setTextArea("")
+        setCancel(true)
+        setSend(false)
+        console.log("Cancel action ON")
+    }
+
+    // Function for Send button ->
+    const handleSend = () => {
+        if (email.trim() && subject.trim() && textArea.trim()) {
+            setSend(true)
+            setCancel(false)
+
+            // Data sending to API
+            fetch("https://email-client-api.dev.io-academy.uk/email", {
+                method: "POST",
+                headers: {
+                    "Content-Type": "application/json"
+                },
+                body: JSON.stringify({
+                    to: email,
+                    subject: subject,
+                    content: textArea
+                })
+            })
+                .then((response) => response.json())
+                .then((data) => {
+                    console.log("Answer API:", data)
+                    setSend(true) // confirmation for sent data
+                })
+                .catch((error) => {
+                    console.error("Error for data sent", error)
+                    setSend(false)
+                })
+        } else {
+            console.log("Complete forms before sending")
```

Comment:
You should display this error message on the page, same way you're displaying the sent/cancelled messages. Please can you word it is 'All fields are required to send an email'